as-ps/as_ps/main.py

```diff
@@ -152,6 +152,12 @@ def write(self, reason, value):
                 ignorestr = ' (1000 events)'
                 _log.info(strf.format(wstr, reason, str(value), ignorestr))
                 self._counter_sofbupdate_cmd = 0
+        elif reason == 'WfmOffset-SP':
+            # suppress printing WfmOfffset setpoints (specially from SOFB)
+            pass
+        #  elif reason == 'IDFFMode-Sel' and value == 1:
```

Comment:
why the different strategy here? for `'WfmOffset-SP'` you didn't comment the `if` and added a `pass` and for `'IDFFMode-Sel'` you commented the `if` statement. Maybe it would be better to adopt the same strategy.

Reply:
this commented line is just in case COMP people did not have time to fix the low-level IDFF which is setting continuously (and unnecessarily) IDFFMode-Sel... but a few moments ago I learned they have a fixed version. I will remove this commented option.